components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -58,7 +58,8 @@ class AddonsManagerAdapter(
     private val addonCollectionProvider: AddonCollectionProvider,
     private val addonsManagerDelegate: AddonsManagerAdapterDelegate,
     addons: List<Addon>,
-    private val style: Style? = null
+    private val style: Style? = null,
+    private val excludedAddons: List<String>? = null
```

Comment:
nit:Let's add some documentation see the other values above.

Reply:
Instead of accepting a null value, the list could be non null `List<String> = emptyList()`